chevah/compat/testing/testcase.py

```diff
@@ -819,12 +825,11 @@ def tearDown(self):
             # an exception here.
             for thread in threads:
                 thread_name = thread.getName()
-                if thread_name == 'MainThread':
-                    continue
-                if thread_name == 'threaded_reactor':
+                if thread_name in self.excepted_threads:
                     continue
-                if thread_name.startswith(
-                        'PoolThread-twisted.internet.reactor'):
+                if ('PoolThread-twisted.internet.reactor' in
+                        self.excepted_threads and thread_name.startswith(
+                        'PoolThread-twisted.internet.reactor')):
```

Comment:
I think that we should allow Contain('PoolThread-twisted.internet.reactor') as part of the list in order to have a more relaxed matching.